### ISSUE TITLE
Irregular numbers

### ISSUE THREAD

**Issue by kolatubosun** [Yorubaname/yorubaname-dashboard#19]
Still haven't figured out why the numbers of the "published" entries in the dashboard is different from that on the homepage.
![screenshot 128](https://cloud.githubusercontent.com/assets/18510601/17528544/cfb1e9d2-5e67-11e6-9738-61a00c1125be.png)
![Uploading Screenshot (129).png…]()

**Comment by temiyemi** [comment]
Yeah, this is strange. Can't figure out why either.

**Comment by kolatubosun** [comment]
Also, today, I realised that a name "Ọláṣení" isn't in the dictionary (though I believe I added it before). Then I re-added it on the front end, and accepted it on the dashboard, by which time it was supposed to go to the "unpublished names" queue. This didn't happen, but it had disappeared from the "suggested name" list. Did it again and got the same result. It's possible that we have a case of "ghost entries" on our hands.
